src/storage/src/hummock/compactor/task_progress.rs

```diff
@@ -40,9 +43,47 @@ impl TaskProgress {
         self.num_ssts_uploaded.fetch_add(1, Ordering::Relaxed);
     }
 
-    pub fn inc_progress_key(&self, inc_key_num: u64) {
-        self.num_progress_key
-            .fetch_add(inc_key_num, Ordering::Relaxed);
+    pub fn inc_num_pending_read_io(&self) {
+        self.num_pending_read_io
```

Comment:
Any reason to use `SeqCst`? If this atomic operation is frequent, there may be some performance issue.

Reply:
This operation is infrequent, but it does not require precise values. I will change it to Relaxed.